src/wp-admin/about.php

```diff
@@ -14,14 +14,14 @@
 /* translators: Page title of the About ClassicPress page in the admin. */
 $title = _x( 'About', 'page title' );
 
-list( $display_version ) = explode( '-', get_bloginfo( 'version' ) );
+$display_version = get_bloginfo( 'classicpress_display_version' );
```

Comment:
I don't think this change is necessary. There's nothing 'WordPress related' here like a prefix and whilst the version number is fetched in a bit of a roundabout way because of the hyphen check for alpha and beta releases, I think we can keep it as is.

Reply:
Here, I'd like us to display the alpha or beta version as is. WP does it this way because their alpha version numbers are very long, but we can do better.

There are a couple more reasons to add `get_bloginfo( 'classicpress_version' )`:

- Introduce a utility method that plugins can use to determine whether they are running WP or CP
- Prepare to change over some uses of `get_bloginfo( 'version' )` later, where appropriate

I'd be fine with adding a new function `classicpress_version` for both of these needs, but right now we don't have a great place to put it. Thoughts?

Reply:
@nylen `wp-includes/functions.php` would be the best place for that.

BTW: AFAIR get_bloginfo also calls get_option, which then again uses apply_filters( 'option_' {option_name} ) .. and thus one automagically is able to adjust a lot of things - if required.

Reply:
I was trying to avoid adding more stuff to this 6000+ line file 😛 I think I'd prefer to split this out into a new include.

`get_bloginfo( 'version' )` just returns `$wp_version` (with a filter applied).

Reply:
To close out this comment thread - I ended up adding a `classicpress_version` function into `src/wp-includes/version.php`.  This file is included in a bunch of different contexts, and we'll want this function to be available in most of them.